src/Components/Assets/AssetImportModal.tsx

```diff
@@ -41,17 +43,39 @@ const AssetImportModal = ({ open, onClose, facility }: Props) => {
     });
   }, []);
 
+  // window["parsePhone"] = parsePhoneNumberFromString;
```

Comment:
```suggestion
  // window["parsePhone"] = parsePhoneNumberFromString;
```
Seems like this can be removed.